snapshot-labs/sx-monorepo
# feat: display formatted bigInt value as compact

### Summary



Closes: #254

Show a more compact voting power value

![Screenshot 2024-04-06 at 21 07 27](https://github.com/snapshot-labs/sx-monorepo/assets/495709/4eaffef3-a7ed-4e0e-b58c-2857dad93cce)

![Screenshot 2024-04-06 at 21 11 04](https://github.com/snapshot-labs/sx-monorepo/assets/495709/293ba8c2-08f3-4c05-af79-554439e077b8)


### How to test

1. Connect to `shot.eth` account via guest.so
2. Go to http://localhost:8080/#/s:safe.eth/proposals
3. It should display a more compact voting power (`2.5M SAFE`) instead of the full number
4. Same thing in the voting power modal (the 'secondary' value still show the full value for more precision)
